packages/use-subscription/package.json

```diff
@@ -15,9 +15,12 @@
   ],
   "license": "MIT",
   "peerDependencies": {
-    "react": "^18.0.0"
```

Comment:
I believe this was accidentally bumped to 18 during the 18 release. So I want to bump it back down. The shim I'm using is compatible with 16.8+.

Reply:
It's a bug in the release process. We have a step that overwrites the versions and it doesn't account for cases where the deps aren't in lockstep with the latest version. Need to add an ad hoc exception here like we do for use-sync-external-store to prevent the build step from writing over it again:

https://github.com/facebook/react/blob/df5d32f230fea7b2ca1e1ddcb78efd6c3f8d7ef2/scripts/rollup/build-all-release-channels.js#L254-L259